test/extended/image_ecosystem/s2i_perl.go

```diff
@@ -45,7 +46,11 @@ var _ = g.Describe("[sig-devex][Feature:ImageEcosystem][perl][Slow] hot deploy f
 
 		g.Describe("hot deploy test", func() {
 			g.It("should work", func() {
-
+				// This image-ecosystem test fails on ARM because it depends on behaviour specific to mod_perl,
+				// which is only included in the RHSCL (RHEL 7) perl images which are not available on ARM.
+				if e2e.NodeOSArchIs("arm64") {
```

Comment:
NodeOSArchIs does not work as you would expect, see #26435 for example.  This needs to be reworked along those lines.
/hold